robot_smach_states/src/robot_smach_states/util/designators/ed_designators.py

```diff
@@ -106,7 +106,8 @@ class EdEntityDesignator(Designator):
 
     def __init__(self, robot, type="", center_point=None, radius=0, id="", parse=True, criteriafuncs=None,
                  weight_function=None,
-                 type_designator=None, center_point_designator=None, id_designator=None, debug=False, name=None):
+                 type_designator=None, center_point_designator=None,
+                 id_designator=None, debug=False, name=None, none_resolve=False):
```

Comment:
- [ ] Please fill lines, before switching to new line
- Resolving to None aka "Not being able to resolve to useful stuff" should be handled correctly in the deepest core of a designator. This really requires a good discussion.